src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/pdl/model/PdlCommon.kt

```diff
@@ -29,10 +29,12 @@ enum class Gradering {
 }
 
 data class PdlPersonNavn(
-        val fornavn: String,
-        val mellomnavn: String?,
-        val etternavn: String
-)
+        var fornavn: String,
+        var mellomnavn: String,
+        var etternavn: String
+) {
+    constructor() : this("", "", "")
+}
```

Comment:
kunne denne funksjonaliteten blitt løst med default verdier = `""` for fornavn, mellomnavn og etternavn?

Reply:
Det har du jaggu rett i.